seatunnel-connectors-v2/connector-common/src/main/java/org/apache/seatunnel/connectors/seatunnel/common/multitablesink/MultiTableWriterRunnable.java

```diff
@@ -66,7 +66,7 @@ public void run() {
                 // exception.
                 throwable = e;
                 break;
-            } catch (Exception e) {
+            } catch (Throwable e) {
```

Comment:
**Issue**: Changing the catch block to catch 'Throwable' instead of 'Exception' could potentially lead to catching and logging 'Error' instances that are meant to be fatal, such as 'OutOfMemoryError', which should not be caught as it can mask serious problems. <br> **Fix**: It's generally safer to catch specific exceptions, such as 'Exception' or more specific types, rather than 'Throwable'. Consider reverting this change unless there's a specific need to catch 'Error' or 'Throwable'. <br> **Code Suggestion**: 
 ```
 -            } catch (Throwable e) {
 +            } catch (Exception e) {
 ```